src/main/java/it/gov/pagopa/payment/controller/payment/CommonPaymentController.java

```diff
@@ -1,16 +1,15 @@
 package it.gov.pagopa.payment.controller.payment;
 
+import it.gov.pagopa.payment.dto.common.BaseTransactionResponseDTO;
 import it.gov.pagopa.payment.dto.qrcode.TransactionCreationRequest;
-import it.gov.pagopa.payment.dto.qrcode.TransactionResponse;
 import jakarta.validation.Valid;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
-
-@RequestMapping("/idpay/payment/")
+@RequestMapping("/idpay/payment")
 public interface CommonPaymentController {
-    @PostMapping("/merchant")
+    @PostMapping("/create-trx")
```

Comment:
review path with @frtrinca 
I think we should let "merchant" somewhere inside the path